diff-analyzer/src/main/resources/org/starchartlabs/chronicler/diff/analyzer/defaultAnalysisSettings.yml

```diff
@@ -0,0 +1,9 @@
+productionFiles:
+   include:
+      - '**/src/**'
+   exclude:
+      - '**/test/**'
+releaseNoteFiles:
+   include:
+      - '**/CHANGE*LOG*'
```

Comment:
Do you think these will cover most standard cases? Right off the top of my head I could see a changelog being called `CHANGES`, which I think this wouldn't catch?

Reply:
OK now you have CHANGES* which would miss CHANGELOG haha. Let's just leave it at CHANGE* how about?

Reply:
I left the original CHANGE*LOG statement in - * is zero or more characters. CHANGE* may grab things like "ChangeSet.java" and be too broad

Reply:
It would, but at some point gotta accept the edge cases. Unless you have an idea how to avoid and still catch the CHANGES file you wanted to catch?
The other way to go is to keep it as it was, and if someone has CHANGES.md or something, they use the YAML configuration

Reply:
Yeah at this point I think just put it back the way you had it :) defaults are defaults, sorry for overthinking it

Reply:
No worries, was good point, but yeah think the original set was a good combination of common cases without being too far-reaching